File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/MessagesItem.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class MessagesItem{

	@SerializedName("mealbarPromoRenderer")
	private MealbarPromoRenderer mealbarPromoRenderer;

	public void setMealbarPromoRenderer(MealbarPromoRenderer mealbarPromoRenderer){
		this.mealbarPromoRenderer = mealbarPromoRenderer;
	}

	public MealbarPromoRenderer getMealbarPromoRenderer(){
		return mealbarPromoRenderer;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/MessagesItem.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class MessagesItem : Parcelable, Serializable {
    @SerializedName("mealbarPromoRenderer")
    var mealbarPromoRenderer: MealbarPromoRenderer? = null
    override fun toString(): String {
        return "MessagesItem{" +
                "mealbarPromoRenderer = '" + mealbarPromoRenderer + '\'' +
                "}"
    }
}